.env.example

```
@@ -0,0 +1,20 @@
+ENV=development
+
+# SMTP Configuration for Email
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USER=your-email@gmail.com
+SMTP_FROM=noreply@bukencoaching.com
+SMTP_TO=coach@bukencoaching.com
+
+# Security
+SESSION_SECRET=change-this-to-a-random-string
+CSRF_KEY=32-byte-key-change-me
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CSRF_KEY key should go before the SESSION_SECRET key

(UnorderedKey)